extensions/community_extensions/skadnetwork.md

```diff
@@ -252,13 +266,13 @@ Used for direct SSP to DSP connections where a DSP wants to only consume their o
         "skadn": {
           "version": "2.0",
```

Comment:
could you please also add `versions` to the example?

Reply:
Done